packages/contracts/contracts/inchain_atomicswap/InchainAtomicSwap.sol

```diff
@@ -192,18 +202,16 @@ contract InchainAtomicSwap is AtomicSwapBase, IInchainAtomicSwap {
             revert MismatchedBidAmount(_bidAmount);
         }
 
-        // TODO: tokenAmount = _bidAmount
         // Calculate the additional token amount being bid
-        uint256 _tokenAmount = _bidAmount - _currentBid.amount;
```

Comment:
This suggested we have to approve <increase> update. 
And then, if we don't do this, the user has already locked the token in the contract side. 
Double lock will happens

Reply:
I think the variable '_currentBid.amount' is always zero here, isn't it?